src-java/kilda-persistence-tinkerpop/src/main/java/org/openkilda/persistence/ferma/repositories/FermaFlowStatsRepository.java

```diff
@@ -74,4 +74,13 @@ protected void doRemove(FlowStatsFrame frame) {
     protected FlowStatsData doDetach(FlowStats entity, FlowStatsFrame frame) {
         return FlowStatsCloner.INSTANCE.deepCopy(frame);
     }
+
+    @Override
+    public void removeByFlowId(String flowId) {
+        framedGraph().traverse(g -> g.V()
+                .hasLabel(FlowStatsFrame.FRAME_LABEL)
+                .has(FlowStatsFrame.FLOW_ID_PROPERTY, flowId))
+                .toListExplicit(FlowStatsFrame.class)
+                .forEach(this::doRemove);
+    }
```

Comment:
Could you please create a unit test for this method?

Reply:
It seems we are testing such logic with integration tests, so, added a couple.